### PR TITLE
fix encoding compat

### PR DESCRIPTION
**Pull request by szha**
## Description ##
Fix encoding compatibility in setup. will create a post release for 0.2.0 post0

### CODE REVIEW

**Comment by szha** [comment]
Fixes #90

**Comment by mli** [comment]
Job PR-91/1 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-91/1/index.html

File: setup.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env python
 import os
-import io
```
Comment (leezu) [inline setup.py]: You are removing the `io` import but start using it below

**Comment by mli** [comment]
Job PR-91/2 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-91/2/index.html

**Pull request merged**